boards/adafruit/feather_esp32s2/doc/adafruit_feather_esp32s2.rst

```diff
@@ -311,14 +309,38 @@ There is a sample available to verify that the MAX17048 fuel gauge on the board
 functioning correctly with Zephyr:
 
 .. note::
-   As of May 31, 2023 Adafruit changed the battery monitor chip from the now-discontinued LC709203
+   As of May 31, 2023 Adafruit changed the battery monitor chip from the now-discontinued LC709203F
    to the MAX17048.
 
 .. zephyr-app-commands::
    :zephyr-app: samples/fuel_gauge/max17048/
    :board: adafruit_feather_esp32s2@C
    :goals: build flash
 
+Testing the Fuel Gauge (LC709203F)
```

Comment:
I would put everything under a "Testing the Fuel Gauge" section with two subsections